internal/index_read.go

```diff
@@ -156,6 +150,9 @@ func readStats(r io.Reader, typ string) (*ReferenceStats, error) {
 	return &stats, nil
 }
 
+// chunkSize determines the number of offsets  consumed by each binary.Read.
```

Comment:
As well? It's only used there AFAICS, so it doesn't need to be here at all.

Also, `vOffs` can be declared as `var vOffs [chunkSize]uint64` below.